src/ccall.cpp

```diff
@@ -1690,6 +1691,17 @@ static jl_cgval_t emit_ccall(jl_codectx_t &ctx, jl_value_t **args, size_t nargs)
         ai.decorateInst(tid);
         return mark_or_box_ccall_result(ctx, tid, retboxed, rt, unionall, static_rt);
     }
+    else if (is_libjulia_func(jl_get_tls_world_age)) {
+        ++CCALL_STAT(jl_get_tls_world_age);
+        assert(lrt == ctx.types().T_size);
+        assert(!isVa && !llvmcall && nccallargs == 0);
+        JL_GC_POP();
+        Instruction *world_age = cast<Instruction>(ctx.world_age_value);
```

Comment:
This still needs to be gated on whether we're in a `toplevel` code block, where the world age is not constant

Reply:
I'm not sure about that because of https://github.com/JuliaLang/julia/blob/4e361b20f439ad696886db917883f23a00f2b10f/src/codegen.cpp#L8986-L8993 which seems to assume it's constant?

Reply:
https://github.com/JuliaLang/julia/blob/4e361b20f439ad696886db917883f23a00f2b10f/src/codegen.cpp#L9635-L9645